.github/workflows/main.yml

```diff
@@ -10,14 +10,31 @@ jobs:
       fail-fast: false
       matrix:
         node-version:
+          - 18
+          - 17
+          - 16
           - 14
           - 12
           - 10
           - 8
+          # Note: not easy to setup GitHub Actions correctly in this case for Node 6, but you can run the tests locally (test-on-all-versions.sh)
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm test
+  test-static-list-is-up-to-date:
```

Comment:
It would be better to do this as a cron job.

Example: https://github.com/sindresorhus/type-fest/commit/ac845e905ef1cb32ffef05e9899a1e4824ee2289

Reply:
Yes, use the same. And yes, I will get a notification then.